src/main/java/core/basesyntax/io/FileReader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.io;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReader {
+    private static final String PATH = "src/main/resources/db/database.csv";
```

Comment:
Are you going to create a new class each time when we need to read a new file from the other location?